apps/dashboard/src/app/(main)/(protected)/projects/[projectId]/users/[userId]/page-client.tsx

```diff
@@ -0,0 +1,140 @@
+      <span className="opacity-75">{icon}</span>
+      <span className="min-w-[160px] font-semibold">{name}</span>
+      <span className="opacity-75">{value}</span>
+    </div>
+  );
+}
+
+type MetadataEditorProps = {
+  value: string,
+}
+
+function MetadataEditor({ value }: MetadataEditorProps) {
+  return <Textarea className="font-mono min-h-[240px]" value={value} readOnly/>;
+}
+
+export default function PageClient() {
+  const stackAdminApp = useAdminApp();
+  const user = stackAdminApp.useUser("753206ff-92b3-4c27-ab11-3b5ec51afafb");
+
+  if (user === null) {
+    return <PageLayout
+      title="User Not Found"
+    >
+      User Not Found
+    </PageLayout>;
+  }
+
+  const name = user.displayName ?? user.primaryEmail ?? 'Anonymous User';
+
+  return (
+    <PageLayout>
+      <div className="flex flex-col gap-4">
+        <div className="flex gap-4 items-center">
+          <Avatar className="w-20 h-20"
+            onClick={() => {
+                console.log(user.toClientJson());
+            }}
+          >
+            <AvatarImage src={user.profileImageUrl ?? undefined} alt={name} />
+            <AvatarFallback>{name.slice(0, 2)}</AvatarFallback>
+          </Avatar>
+          <div>
+            <h1 className="text-3xl font-semibold">{name}</h1>
+            <p>Last active {fromNow(user.lastActiveAt)}</p>
+          </div>
+        </div>
+        <div className="flex flex-col gap-2 text-sm p-4">
+          <UserInfo icon={<Hash size={16}/>} name="User ID" value={user.id}/>
+          <UserInfo icon={<Mail size={16}/>} name="Primary email" value={user.primaryEmail ?? '-'}/>
+          <UserInfo icon={<AtSign size={16}/>} name="Display name" value={user.displayName ?? '-'}/>
+          <UserInfo icon={<SquareAsterisk size={16}/>} name="Password" value={user.hasPassword ? '************' : '-'}/>
+          <UserInfo icon={<Shield size={16}/>} name="2-factor auth" value={user.otpAuthEnabled ? 'Enabled' : 'Disabled'}/>
+          <UserInfo icon={<Calendar size={16}/>} name="Signed up at" value={user.signedUpAt.toDateString()}/>
+        </div>
+      </div>
+      {/* <Card>
+        <CardHeader>
+          <CardTitle>
+            Social Accounts
+          </CardTitle>
+        </CardHeader>
+        <CardContent>
+          <div className="flex flex-col gap-2">
+            <div className="flex items-center gap-2 opacity-75">
+              <img src="https://github.com/favicon.ico" className="w-4 h-4" alt="GitHub"/>
+              <span>GitHub</span>
+              <span className="text-sm">Connected as @thecactusblue</span>
+            </div>
+            <div className="flex items-center gap-2 opacity-75">
+              <img src="https://google.com/favicon.ico" className="w-4 h-4" alt="Google"/>
+              <span>Google</span>
+              <span className="text-sm">Connected as example@gmail.com</span>
+            </div>
+          </div>
+        </CardContent>
+      </Card> */}
```

Comment:
don't leave large code comments (you can always get them from the git history later)